agent/lib/kontena/workers/node_info_worker.rb

```diff
@@ -24,7 +24,7 @@ def initialize(queue, autostart = true)
       @statsd = nil
       subscribe('websocket:connected', :on_websocket_connected)
       subscribe('agent:node_info', :on_node_info)
-      info 'initialized'
+      debug 'initialized'
```

Comment:
This should stay as `info`, it only happens once at startup and then again after a worker crash.